lib/phoenix/live_dashboard/layout_view.ex

```diff
@@ -2,8 +2,11 @@ defmodule Phoenix.LiveDashboard.LayoutView do
   @moduledoc false
   use Phoenix.LiveDashboard.Web, :view
 
-  js_path = Path.join(__DIR__, "../../../priv/static/js/app.js")
-  css_path = Path.join(__DIR__, "../../../priv/static/css/app.css")
+  js_bundle = if Mix.env() == "prod", do: "app.js", else: "app.dev.js"
+  css_bundle = if Mix.env() == "prod", do: "app.css", else: "app.dev.css"
```

Comment:
Just to be clear, this means we will only get source map when running in development locally. If that's the case, maybe we should generate the dev bundle in a tmp directory so we don't include it in the package? 🤔

Reply:
To give a bit more context, packages always run `Mix.env() == "prod"`. If the goal is only for local development, then I think the approach in #158 is simpler but I believe you also want source maps when used as a dep. Unfortunately though, there is no way to achieve it besides making it opt-in somehow.